Wrong import at our lib/index.js files for internal components

Should:
All our components should define the correct dependencies at lib folder. 

Is:
For example "Modal": at lib/index.js there is this line of code:
`import closeIcon from '@axa-ch/materials/icons/material-design/close.svg';`

And because of that i am getting this error:
`index.js?b573:1 Uncaught Error: Cannot find module '@axa-ch/materials/icons/material-design/close.svg'`

And this is my package.json:
```
"@axa-ch/materials": "^13.0.0",
    "@axa-ch/modal": "^1.0.1",
``` 

The problem:
Materials v13 do not have the folder "material-design". But the Modal is referencing it.

--- a/src/components/30-organisms/modal/CHANGELOG.md
+++ b/src/components/30-organisms/modal/CHANGELOG.md
@@ -1,3 +1,7 @@
+## 1.0.2
+
+- Fix: Modal did not have `@axa-ch/materials` as dependency.
+
 ## 1.0.1
 
 - Fix: close modal in Safari by pressing outside the modal.

--- a/src/components/30-organisms/modal/package.json
+++ b/src/components/30-organisms/modal/package.json
@@ -26,6 +26,7 @@
     "url": "https://github.com/axa-ch/patterns-library/issues"
   },
   "dependencies": {
+    "@axa-ch/materials": "14.0.0",
     "@skatejs/val": "^0.4.0",
     "lit-element": "^2.2.0",
     "lit-html": "^1.1.0"

--- a/src/components/30-organisms/modal/package.json
+++ b/src/components/30-organisms/modal/package.json
@@ -26,7 +26,7 @@
     "url": "https://github.com/axa-ch/patterns-library/issues"
   },
   "dependencies": {
-    "@axa-ch/materials": "14.0.0",
+    "@axa-ch/materials": "^14.0.0",
     "@skatejs/val": "^0.4.0",
     "lit-element": "^2.2.0",
     "lit-html": "^1.1.0"

